Server.Tests/ParserTests.cs

```diff
@@ -89,5 +89,70 @@ public void Request_With_Body_Should_Return_Correct_Body()
             Assert.AreEqual("1.1", actual.Version);
             Assert.AreEqual("THIS IS THE BODY", actual.Body);
         }
+
+
+        [TestMethod]
+        public void Requests_With_Specific_Method_Should_Return_Correct_Method_Value_Test()
+        {
+            // 1. Arrange
+            RequestParser parser = new RequestParser();
+            var GETrequestString = @"GET /one/two?three=4 HTTP/1.1";
```

Comment:
These are 6 different tests, so write 6 different methods. A test should have a single mode of failure - what information do you have if `Requests_With_Specific_Method_Should_Return_Correct_Method_Value_Test` reports failure?

Reply:
I would probably get information about the first mode of failure that appeared, which might or might not be the only one.
Therefore  I guess and agree that a test should have a single reason to fail most of the time.

Thank you, the changes have been applied.